lib/board.rb

```diff
@@ -55,12 +57,12 @@ def diagonals
 
   def position_to_row(position)
     zero_based_position = position.to_i - 1
-    (zero_based_position / 3).ceil # 3 is number of cols
+    (zero_based_position / board.length).ceil
```

Comment:
isn't `board.length` the same as `board.width`? is there a specific reason to use one not the other? (here, and in Strategy class as well)

Reply:
There isn't a real different. I will change it! 

I was a little confused about what you meant by "in Strategy class as well". Could you let me know where you were looking at?

Reply:
I was just saying that board.width is used not only in this class but in Strategy as well (maybe somewhere else too), and if it is the same as board.length it would be nice to have it consistent everywhere

Reply:
got it! I will use `width` everywhere, thanks!